src/PlacesAutocomplete.js

```diff
@@ -282,6 +290,8 @@ PlacesAutocomplete.propTypes = {
 };
 
 PlacesAutocomplete.defaultProps = {
+  clearItemsOnError: false,
+  onError: () => console.error('place autocomplete failed'),
```

Comment:
This error message looks good to me 👍 The only thing I would possibly change is putting the status on a new line for readability.